native/test/src/tests/auth.rs

```diff
@@ -288,7 +286,7 @@ async fn can_register_via_email() -> Result<()> {
         resp.submit_url(),
     );
 
-    read_email_msg("test2", "test", "_matrix/client/unstable/registration").await?;
+    confirm_email_msg(email, "_matrix/client/unstable/registration").await?;
```

Comment:
Ah, you did not only read email msg, but also submit there?
Great.

Reply:
submitting was already there... the code on `read` would also do this before, it wasn't really clear though.